src/online-members/utils.js

```diff
@@ -0,0 +1,65 @@
+async function makeApiCall(
+  url,
+  method = 'get',
+  body = null,
+  headers = [],
+  options = null,
+) {
+  try {
+    const response = await fetch(url, {
+      method,
+      body,
+      headers,
+      ...options,
+    });
+    return response;
+  } catch (err) {
+    console.error('Something went wrong. Please contact admin', err);
```

Comment:
This is something , we would be using in many places, would it be better to keep it in constants?